src/olympia/versions/views.py

```diff
@@ -98,14 +99,24 @@ def is_appropriate_reviewer(addon, channel):
 
     attachment = bool(type == 'attachment')
     if use_cdn:
-        # When using the CDN URL, we do a redirect, so we can't set
-        # Content-Disposition: attachment for attachments. To work around this,
-        # if attachment=True, get_file_cdn_url() changes the path to something
-        # we recognize in the nginx config.
-        loc = urlparams(file_.get_file_cdn_url(attachment=attachment),
-                        filehash=file_.hash)
-        response = http.HttpResponseRedirect(loc)
-        response['X-Target-Digest'] = file_.hash
+        # When serving the file for the general public through the CDN, we need
+        # to obey regional restrictions
+        region_code = request.META.get('HTTP_X_COUNTRY_CODE', None)
```

Comment:
It's interesting that we call the header "country code" but the variable is "region code".

Reply:
Yeah. We called the header country code because it follows https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2 but I didn't want to call the model/field country because it was a bit of an oversimplification IMHO - technically, a different country code would be sent for special collectivities and territories, even if it's the same "country".

For instance, Guadeloupe has its own ISO 3166-1 alpha-2 country code, despite being part of France.